### ISSUE TITLE
Cannot upload code onto cortex

### ISSUE THREAD

**Issue by Aponthis** [purduesigbots/pros-cli2#28]
#### Expected Behavior: 
Upload of code to cortex

#### Actual Behavior: 
Does not upload (duh)
```
Traceback (most recent call last):
  File "\<frozen importlib._bootstrap>", line 961, in _find_and_load
  File "\<frozen importlib._bootstrap>", line 950, in _find_and_load_unlocked
  File "\<frozen importlib._bootstrap>", line 646, in _load_unlocked
  File "\<frozen importlib._bootstrap>", line 616, in _load_backward_compatible
  File "C:\Jenkins\workspace\rduesigbots_pros-cli_master-XDC5XVNHR2J6WR2WUV4MI7AFO7ADKVMGTOQFTSEJTYA4Z56HENRA\.virtualenv\lib\site-packages\cx_Freeze\initscripts\__startup__.py", line 12, in <module>
  File "\<frozen importlib._bootstrap>", line 961, in _find_and_load
  File "\<frozen importlib._bootstrap>", line 950, in _find_and_load_unlocked
  File "\<frozen importlib._bootstrap>", line 646, in _load_unlocked
  File "\<frozen importlib._bootstrap>", line 616, in _load_backward_compatible
  File "C:\Jenkins\workspace\rduesigbots_pros-cli_master-XDC5XVNHR2J6WR2WUV4MI7AFO7ADKVMGTOQFTSEJTYA4Z56HENRA\.virtualenv\lib\site-packages\cx_Freeze\initscripts\Console.py", line 24, in <module>
  File "proscli/main.py", line 44, in <module>
  File "proscli/main.py", line 12, in main
  File "C:\Jenkins\workspace\rduesigbots_pros-cli_master-XDC5XVNHR2J6WR2WUV4MI7AFO7ADKVMGTOQFTSEJTYA4Z56HENRA\.virtualenv\lib\site-packages\click\core.py", line 697, in main
  File "C:\Jenkins\workspace\rduesigbots_pros-cli_master-XDC5XVNHR2J6WR2WUV4MI7AFO7ADKVMGTOQFTSEJTYA4Z56HENRA\.virtualenv\lib\site-packages\click\core.py", line 1066, in invoke
  File "C:\Jenkins\workspace\rduesigbots_pros-cli_master-XDC5XVNHR2J6WR2WUV4MI7AFO7ADKVMGTOQFTSEJTYA4Z56HENRA\.virtualenv\lib\site-packages\click\core.py", line 895, in invoke
  File "C:\Jenkins\workspace\rduesigbots_pros-cli_master-XDC5XVNHR2J6WR2WUV4MI7AFO7ADKVMGTOQFTSEJTYA4Z56HENRA\.virtualenv\lib\site-packages\click\core.py", line 535, in invoke
  File "C:\Jenkins\workspace\rduesigbots_pros-cli_master-XDC5XVNHR2J6WR2WUV4MI7AFO7ADKVMGTOQFTSEJTYA4Z56HENRA\.virtualenv\lib\site-packages\click\decorators.py", line 64, in new_func
  File "C:\Jenkins\workspace\rduesigbots_pros-cli_master-XDC5XVNHR2J6WR2WUV4MI7AFO7ADKVMGTOQFTSEJTYA4Z56HENRA\.virtualenv\lib\site-packages\click\core.py", line 535, in invoke
  File "C:\Jenkins\workspace\rduesigbots_pros-cli_master-XDC5XVNHR2J6WR2WUV4MI7AFO7ADKVMGTOQFTSEJTYA4Z56HENRA\proscli\flasher.py", line 43, in flash
  File "C:\Jenkins\workspace\rduesigbots_pros-cli_master-XDC5XVNHR2J6WR2WUV4MI7AFO7ADKVMGTOQFTSEJTYA4Z56HENRA\prosflasher\ports.py", line 16, in list_com_ports
  File "C:\Jenkins\workspace\rduesigbots_pros-cli_master-XDC5XVNHR2J6WR2WUV4MI7AFO7ADKVMGTOQFTSEJTYA4Z56HENRA\prosflasher\ports.py", line 16, in <listcomp>
AttributeError: 'NoneType' object has no attribute 'lower'
```

#### Steps to reproduce: 
This is the command entered into PowerShell to get the output, which has the same message as clicking the upload button:  `pros flash -f "C:\Users\Lemon Bots\.atom\127C BM"`

I just installed Atom and installed the PROS package through Atom. I have Python 3.6 installed as well.
#### System information:
Operating System:  Windows 10 Home

PROS Version: 0.65

#### Additional Information
N/A
#### Screenshots/Output Dumps/Stack Traces
When I try to upload through Make upload:

![image](https://cloud.githubusercontent.com/assets/13190468/23335459/181928cc-fb73-11e6-8dd8-d1545ea6d900.png)

**Comment by Aponthis** [comment]
The `Make upload` issue was due to a missing file, and when that was remedied, not having Java installed. That can be disregarded.

**Comment by HotelCalifornia** [comment]
Can you clarify?
Did you not have Java installed, and then installing it fixed the issue? Or did you mean uninstalling Java was the fix?

**Comment by Aponthis** [comment]
Installing Java fixed the second issue, the one in the screenshot.

**Comment by jwalk511** [comment]
From Discord: The project was generated on Pros for Eclipse. It was then brought over to a new machine with Pros-Atom that did not have Java. After putting `libpros.a` inside `firmware` and installing Java, `make upload` works as expected.

**Comment by Aponthis** [comment]
Note: the upload button still does not work, and gives the lengthy error message in the above code section.

**Comment by jabibi100** [comment]
If you create a new project within Atom can you flash it to the cortex utilizing the upload button without issue?

**Comment by HotelCalifornia** [comment]
@Aponthis are you still having this issue?

**Comment by Aponthis** [comment]
@HotelCalifornia I can upload through pros make upload . I haven't tried the upload button (can't remember the command equivalent). So everything is fine insofar as I can upload but I can try using the upload button again and see if the issue is still there for you.

**Comment by HotelCalifornia** [comment]
That would be great, thanks.

The CLI command the button runs should be `pros flash`.

If you're still getting errors with that, I'd say wait for the fixes in #27 to get released (which should be soon), and see if that fixes your issues.
